modules/outbound-domain-event-handler/src/application/handlers/bulk_transfers_callback_received.ts

```diff
@@ -19,8 +19,8 @@ export async function handleBulkTransfersCallbackReceived(
             bulkId: bulkTransfersCallbackReceivedMessage.getKey(),
             content: {
                 batchId: bulkTransfersCallbackReceivedMessage.batchId,
-                bulkTransferId: bulkTransfersCallbackReceivedMessage.bulkTransferId,
```

Comment:
You are going to need this in the `bulkTransfersErrorResult` <-- it does not contain the bulkTransferId, or am I missing something?

Reply:
`bulkTransferId` is part of a `bulkTransfersResult` (`BulkTransferResponse` type)